CONTRIBUTING.md

```diff
@@ -121,6 +121,16 @@ this:
 You can format individual files in place by running `clang-format -i <file>...`
 from any directory within this project.
 
+There is a Continuous Integration job that runs clang-format on PRs. If the code does not comply, an artifact will be generated which is a patch file that corrects any auto-fixable formatting issues.
+
+To download the patch file:
+
+1. Next to `clang-format / check (pull_request) Failing after #s` -> click **Details** to open the details page.
+2. Left menu -> click **Summary**
```

Comment:
```suggestion
2. Click **Summary** from the left menu.
```

Reply:
Similarly - you generally need to look for the left menu before you can click **Summary**.